Rakefile

```diff
@@ -309,10 +313,9 @@ REGEN_METADATA = {
         tag: 'arm_recovery'
     },
     azure_mgmt_recovery_services_backup: {
-        spec_uri: 'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-recoveryservicesbackup/compositeRecoveryServicesBackupClient.json',
+        spec_uri: 'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-recoveryservicesbackup/readme.md',
```

Comment:
I assume this is the only one that has a readme.md right now, so should be fine to just update this one.